config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-v1beta1-release-1.9.yaml

```diff
@@ -1,9 +1,85 @@
 periodics:
-- name: periodic-cluster-api-provider-azure-capi-e2e-v1beta1-release-1-7
+- name: periodic-cluster-api-provider-azure-conformance-v1beta1-release-1-9
   decorate: true
   decoration_config:
     timeout: 4h
-  interval: 72h
+  interval: 24h
```

Comment:
I think the release-1.7 file needs to change to release-1.8 and release 1.8 to release-1.9 so we can keep the right intervals and tests for both (older release runs periodic less often)

Reply:
Oh, good call. I copied the 1.8 version and changed values to 1.9, but you're correct: will update.